### ISSUE TITLE
Reference "level of git" from centralized collaborative workflow?

### ISSUE THREAD

**Issue by juholehtonen** [coderefinery/git-intro#173]
Would it be useful to reference the level of git lesson https://coderefinery.github.io/git-intro/14-level/ from "Collaborative distributed version control: Working with remotes and centralized workflow" (https://github.com/coderefinery/git-collaborative/blame/gh-pages/_episodes/01-remotes.md#L250)?

**Comment by bast** [comment]
Good idea but this is then in the wrong repo. Moving this issue to git-collaborative.
